lib/shp-write-stream/index.js

```diff
@@ -143,6 +144,6 @@ class Input extends Writable {
   }
 }
 
-module.exports = function (schema, createStream, cb) {
-  return new Input(schema, createStream ,cb);
+module.exports = function (schema, createStream, cb, enc) {
```

Comment:
`cb` at the end is a Node.js convention.
We could rewrite the function signature like this: `function (schema, createStream, options, cb)` and do the same thing as Input constructor to avoid a breaking change.